packages/insomnia-inso/src/commands/lint-specification.js

```diff
@@ -15,13 +18,33 @@ export async function lintSpecification(
 
   const specFromDb = identifier ? loadApiSpec(db, identifier) : await promptApiSpec(db, !!ci);
 
-  if (!specFromDb) {
-    logger.fatal(`Specification not found.`);
+  let specContent: string = '';
+
+  try {
+    if (specFromDb?.contents) {
+      logger.trace('Linting specification from database contents');
+      specContent = specFromDb.contents;
+    } else if (identifier) {
+      // try load as a file
+      const fileName = path.isAbsolute(identifier)
+        ? identifier
+        : path.join(workingDir || '.', identifier);
+      logger.trace(`Reading specification from file \`${fileName}\``);
+      try {
+        specContent = (await fs.promises.readFile(fileName)).toString();
+      } catch (e) {
+        throw new InsoError('Specification not found.');
```

Comment:
Please update this error message to the following, or something similar:
```suggestion
        throw new InsoError(`Failed to read "${fileName}"`, e);
```